calico_cni_k8s_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
I think you should be able to get rid of the `"log"` import and use `logrus` everywhere and import it (logrus) as `log` to be consistent with other files